main/src/main/scala/sbt/internal/Idle.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt.internal
+
+import scala.concurrent.duration._
+import scala.util.control.NonFatal
+
+private[sbt] object Idle {
+  val value: FiniteDuration =
+    try System.getProperty("sbt.idle.seconds", "300").toInt.seconds
+    catch { case NonFatal(_) => 5.minute }
+  val idleTimeout: FiniteDuration = 500.millis
```

Comment:
So in the worst case, it will add 0.5 second latency after you type `test` and hit the enter key?

Reply:
Yeah.

Reply:
Could we reduce the latency to somewhere low enough that human would not perceive it but high enough that CPU won't mind?